metastore/scripts/upgrade/derby/045-HIVE-16886.derby.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO "APP"."NOTIFICATION_SEQUENCE" ("NNI_ID", "NEXT_EVENT_ID") SELECT * FROM (VALUES (1,1)) tmp_table WHERE NOT EXISTS ( SELECT "NEXT_EVENT_ID" FROM "APP"."NOTIFICATION_SEQUENCE");
```

Comment:
I think the SELECT * FROM (VALUES(1,1,)) can be replace by a simple SELECT 1,1

I tested the SELECT 1,1; on Mysql. Would it work on the rest of the DBs?

Reply:
not it would not work on all dbs, there has to be different sql semantics for all of them which are tested on each one of them